src/services/share-buttons.service.ts

```diff
@@ -49,6 +49,11 @@ export class ShareButtonsService {
             args = Object.assign({}, args, { via: this.twitterAccount });
         }
 
+        /** check for mobile browser (this won't detect tablets browser) */
+        if (this.window.innerWidth <= 480) {
```

Comment:
I think an utilty method `isMobile()` as this one, in `WindowService`, is way better than the above:

```
export class WindowService {
...

    isMobile(){
         return /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(this.nativeWindow().navigator.userAgent);
     }
}
```

For a more complete list of mobile browsers, see [detectmobilebrowsers.com](http://detectmobilebrowsers.com/)